edweiss-app/components/lectures/slides/StudentQuestion.tsx

```diff
@@ -18,9 +22,10 @@ const StudentQuestion: ReactComponent<{ courseName: string, lectureId: string, q
     const [question, setQuestion] = useState<string>('');
     const [isLoading, setIsLoading] = useState<boolean>(false);
     const [isAnonym, setIsAnonym] = useState<boolean>(false);
-    const [enableDisplay, setEnableDisplay] = useState<boolean>(false);
     const { uid } = useAuth();
 
+    const sortedQuestions = useDynamicDocs(CollectionOf<LectureDisplay.Question>(`courses/${courseName}/lectures/${lectureId}/questions`).orderBy("likes", "desc"));
+
```

Comment:
It makes sens to order them by likes. That's a great insight, so that the audience can focus there attention on the most liked questions. Good job.

Reply:
Yes indeed it is more valuable this way